README.md

```diff
@@ -0,0 +1,199 @@
+framework to the current
+[testharness.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/testharness.js)
+framework.
+
+
+View the design document at [go/web-test-transformer](http://go/web-test-transformer) (internal Google access 
+only).
+
+## Setup
+
+`$git clone git@github.com:Ecosystem-Infra/web-test-transformer.git`
+
+`$cd web-test-transformer`
+
+`$npm install` to install dependencies.
+
+## Usage
+
+Requirements:
+- You are **not** running the tool on Windows (no path separator support, see https://github.com/Ecosystem-Infra/web-test-transformer/issues/11). 
+- You must be in `chromium/src/third_party/blink/web_tests` within the Chromium source tree (for verification)
+- You must have content_shell built. (for verification)
+
+
+Here are the high level steps for using the tool to update tests:
+1. Run the tool on some tests
+2. Make note of the summary output: particularly transformation and verification failures
+3. Manually review: restore failures, make adjustments on tests
+4. Commit any transformations you wish to keep.
+5. Send a CL for review and CQ run
+
+
+### Dry Run
+By default, the tool overwrites the original file and changes can be viewed using `git diff`.
+Sometimes that output can be hard to read and understand the full context.
+If you would like to write the transformed file to a different path to compare the files, 
+create a directory and pass the path to the `--output_dir` flag. The transformed files will
+appear in that directory with their original names.
+
+**Dry Run Example**
+
+`$mkdir transformed_temp`
+
+`$node ~/web-test-transformer/index.js --dir=./fast/dom/SelectorAPI --output_dir=./transformed_temp`
+
+*review the transformations or diffs:*
+`$diff ./fast/dom/SelectorAPI/caseID.html ./transformed_temp/caseID.html`
+
+`$rm -r transformed_temp`
+
+### CLI Flags
+`$node index.js --help`
+
+```
+Options:
+  --file: Path to test file to transform
+    (default: null)
+  --dir: Path to dir of test files to transform
+    (default: null)
+  --output_dir: Path to dir where output files should be written. If 
+    null, will overwrite input files. 
+    (default: null)
+  --target_build: Target build used in -t parameter for run_web_tests.py
+    (default: "Default")
+  --[no]quiet: Disable logging
+    (default: false)
+  --[no]verify: Runs web test after transforming.
+    (default: true)
+```
+
+#### Required Flags
+You must specify **exactly one** of `--file` or `--dir`.
+
+### Examples
+
+Transforming a directory of tests, running verification with a build called out/Debug
+
+`$node ~/web-test-transfomer/index.js --dir=fast/dom/SelectorAPI --target_build=Debug`
+
+Transforming a single file fast/files/file-list-test.html, skipping verification 
+
+`$node ~/web-test-transformer/index.js --file=fast/files/file-list-test.html --noverify`
+
+**Example Output:**
```

Comment:
```suggestion
<details><summary>**Example Output:**</summary>
```

Let's collapse the example outputs.

Reply:
This is really cool, didn't know you could do this